src/stores/MessageStore.js

```diff
@@ -9,17 +9,6 @@ let CHANGE_EVENT = 'change';
 
 let _messages = {};
 
-function _addMessages(rawMessages) {
```

Comment:
@rishiraj824 I am still not able to find the usage of `_addMessages` in actions.js, can you point me to the line please?